attest

```diff
@@ -1,16 +1,16 @@
 #!/usr/bin/env python3
-## ---------------------------------------------------------------------
-##
-## Copyright (c) 2019 - 2020 by the IBAMR developers
-## All rights reserved.
-##
-## This file is part of IBAMR.
-##
-## IBAMR is free software and is distributed under the 3-clause BSD
-## license. The full text of the license can be found in the file
-## COPYRIGHT at the top level directory of IBAMR.
-##
-## ---------------------------------------------------------------------
+# ---------------------------------------------------------------------
+#
+# Copyright (c) 2019 - 2020 by the IBAMR developers
```

Comment:
Do we still need to update to 2021?

Reply:
the update copyright release step will take care of that.